### DISCUSSION TITLE
Desabilitar sincronização de imagens, histórico na pasta src/storages

### DISCUSSION THREAD

**Discussion by iagocotta** [aldinokemal/go-whatsapp-web-multidevice#200] [General]
Primeiramente gostaria de parabenizar pela ferramenta fantástica que desenvolveu, muito rápida!

Gostaria de saber se é possível desabilitar a sincronização de imagens, histórico etc após Efetuar o Login e ler o QRCODE, pois meu objetivo é utilizar a ferramenta apenas como disparador de mensagens e anexo (PDF) via API, logo não tem necessidade de ficar sincronizando mensagens do whatsapp, imagens etc. 

Notei que mesmo sem enviar nada fica adicionando na pasta as imagens etc, conforme print abaixo:

![image](https://github.com/user-attachments/assets/38a40ac7-a58c-4904-98ed-32ecd8af766b)

Isso é possível de desabilitar?  
Obrigado!

**Comment by aldinokemal** [comment]
It seems possible. Can you customize here ref link https://github.com/aldinokemal/go-whatsapp-web-multidevice/blob/661a7cec64c6efdeb5b7c86dd2481de84bf063b3/src/pkg/whatsapp/whatsapp.go#L490
